sdk/digitaltwins/azure-digitaltwins-core/src/samples/java/com/azure/digitaltwins/core/helpers/FileHelper.java

```diff
@@ -31,7 +33,13 @@
                     String fileAsString = lines.collect(Collectors.joining());
                     lines.close();
 
-                    fileContents.put(getFileNameFromPath(filePath), cleanupJsonString(fileAsString));
+                    if (clazz == String.class) {
+                        // don't deserialize into a type
+                        fileContents.put(getFileNameFromPath(filePath), (T) cleanupJsonString(fileAsString));
+                    }
```

Comment:
Confused about Java. Should it be `} else {` (all on the same line)?

Reply:
It should be one line. I'll change it